basic_pipeline_extension/01.0_Bin.md

```diff
@@ -0,0 +1,106 @@
+# Bin
+
+A Membrane's bin is a container for elements, which allows for creating reusable groups of elements.
+Bin is similar to a pipeline in that it consists of linked elements. Such bin can then be placed inside a pipeline and linked with other elements. Bins can also be nested within one another.
+Bin also has another advantage - it manages its children, for instance by dynamically spawning or replacing them as the stream changes.
+
+# Enclosing pipeline elements inside a bin
+
+As you can see, we have `Source` -> `Ordering Buffer` -> `Depayloader` chain, which is duplicated.
+![Pipeline scheme](../basic_pipeline/assets/images/basic_pipeline.png) <br>
+
```

Comment:
Shouldn't we change that image too, so that it has "A", "B" and "AB" tags in the file fields?

Reply:
Yes, however, we are referencing the basic_pipeline/ folder here, so once merged this image will update.
Maybe it's better to duplicate the image and put it inside the basic_pipeline_extension folder so that all assets for this guide are in one place? WDYT?

Reply:
Oh, I see. I think we should duplicate it, so that there are no references among directories. 
However, in case we needed to share some assets more frequently, we can think about creating the assets/ directory in the repository's root catalog.